src/IB/IBFEMethod.cpp

```diff
@@ -885,8 +693,7 @@ IBFEMethod::backwardEulerStep(const double current_time, const double new_time)
             TBOX_ERROR("IBFEDirectForcingKinematics does not implement backwardEulerStep().\n");
         }
     }
-    return;
-} // backwardEulerStep
+}
```

Comment:
Could we please avoid these types of code updates in this PR? I like this change but its making this patch bigger than it needs to be.

Reply:
I knew you would say this, but let me tell you why I went ahead and did it: I am now running `clang-tidy` interactively, and getting rid of these `return` statements makes `clang-tidy` happy. (And I figured if I am wiping out the unneeded `return` I also might as well get rid of the unneeded comment, which dates back to days when I did not have good block matching tools in my editor.)